examples/data-objects/inventory-app/src/reactSharedTreeView.tsx

```diff
@@ -11,13 +11,13 @@ import { IChannelFactory } from "@fluidframework/datastore-definitions";
 import {
```

Comment:
Be aware that this code moved to its own project, so you will have to resolve that when merging in main.